src/cljs/replumb/repl.cljs

```diff
@@ -827,7 +853,14 @@
   file is found). It is mutually exclusive with :load-fn! and will be
   ignored in case both are present
 
-  * :src-paths - a vector of paths containing source files
+  * `:write-file-fn!` a synchronous 2-arity function that accepts a file-path
+  and data to write.
+
+  * `:src-paths` - a vector of paths containing source files
+  * `:cache-path` - if present, will indicate the path of the cached files
+  * `:search-cache-in-src-paths?` - if true, will look for cached files in
```

Comment:
Well, in theory we have 4 possibilities, as shown in the table below:
cache folder | search cache in `src-path`?
--- | ---
0 | 0
0 | 1
1 | 0
1 | 1